uaparser/parser.go

```diff
@@ -222,7 +229,7 @@ func NewFromSaved() *Parser {
 func NewFromBytes(data []byte) (*Parser, error) {
```

Comment:
```suggestion
func NewFromBytes(data []byte) (*Parser, error) {
  return newFromBytes(data, cDefaultCacheSize)
}
 
func newFromBytes(data []byte, parserCacheSize) (*Parser, error) {
```
continuing from my above suggestion